electron-app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-list-item
+    v-if="amount !== zero"
+    :id="`${name}_box`"
+    :ripple="false"
+    class="blockchain-balance-box__item"
+    @click="doNothing"
+  >
+    <v-list-item-avatar
+      tile
+      class="blockchain-balance-box__icon blockchain-box__icon"
+    >
+      <crypto-icon
+        width="24px"
+        :symbol="blockchainBalanceIcons[name]"
+      ></crypto-icon>
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title style="display: flex; justify-content: space-between;">
```

Comment:
is this done to ovewrite the `v-list-item-title` style?

Reply:
It would need to be done regardless of what element is used there so that content is centered vertically and horizontally. It can go in a class instead of a style if that's what you mean? I'm trying to figure out if there are vuetify helper classes that we could use instead.

Reply:
@kelsos @isidorosp is this comment handled? Can we consider it resolved?

Reply:
Yes, no inline styles anymore.